README.md

```diff
@@ -1,65 +1,22 @@
 # Hello Java Mesh
-This is the simplest example app using the java version of the RightMesh 
+This is the simplest example app using the java version of the RightMesh
 library. It is meant to operate on Linux / Windows / Raspberry PI etc,
-which support java. This app can interoperate with the Android version:
+or any other platform supporting Java.
+This app can interoperate with the Android version:
 https://github.com/RightMesh/HelloMesh
 
 ## Adding the RightMesh library
-Update the [gradle.build](build.gradle) file with the following:
-```applicationDefaultJvmArgs = ["-noverify"]
-buildscript {
-    ext {
-        artifactory_app_username = "insertyourusername"
-        artifactory_app_password = "insertyourpasswordhash"
-        artifactory_app_key = "insertyourkey"
-    }
-    repositories {
-        maven {
-            url "http://research.rightmesh.io/artifactory/libs-local"
-            credentials {
-                username artifactory_app_username
-                password artifactory_app_password
-            }
-        }
-        mavenCentral()
-        jcenter()
-    }
-    dependencies {
-        classpath 'io.left.rightmesh:rightmesh-plugin:1.3'
-    }
-}
-apply plugin: 'io.left.rightmesh.rightmesh-plugin'
-build.dependsOn("rightmesh")
+Refer to [build.gradle](build.gradle) for an example of integrating
+RightMesh into your own application.
 
-repositories {
-    mavenCentral()
-    jcenter()
-    maven {
-        url "http://research.rightmesh.io/artifactory/libs-local"
-        credentials {
-            username artifactory_app_username
-            password artifactory_app_password
-        }
-    }
-}
-
-dependencies {
-    compile ('io.left.rightmesh:rightmesh-java-library:0.4.0')
-}
-```
-
-## Building
+## Using this app
 `./gradlew installDist`
-
-## Running
 `./gradlew run` or `./build/install/HelloJavaMesh/bin/HelloJavaMesh`
 
 ## Notes on running
```

Comment:
Looks like this paragraph is super out of date. Maybe a quick preface like this could work (should be able to auto-approve it if you're fine with it):

```suggestion
## Notes on running

If you have an Internet connection, HelloJavaMesh will connect to a RightMesh Superpeer and discover peers around the world who are also connected to a superpeer.

```

Reply:
Ah, I see it had a line at the end that was removed.

Reply:
Might be good to clear up that last paragraph to something like

>>>At this time, the Java library does not automatically join the *local mesh
network* so it is up to the developer to manually join the network with the device
the Java code is running on. For testing purposes you should be able to
join any *WiFi network being broadcast by a phone running RightMesh* with *an
SSID following the pattern `RM-XXXXXX` using* the password

Reply:
Okay, I'll add that to the other PR.